cms/envs/common.py

```diff
@@ -1212,3 +1222,8 @@
 # Used with Email sending
 RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS = 5
 RETRY_ACTIVATION_EMAIL_TIMEOUT = 0.5
+
+############## DJANGO-USER-TASKS ##############
+
+USER_TASKS_ARTIFACT_EXPIRE = 60 * 60  # 1 hour
+USER_TASKS_MAX_AGE = timedelta(days=7)
```

Comment:
Actually, I'll also remove that first setting until we actually implement storage of artifacts for export tasks; might want to rename it or change the value type then anyway